wp-content/plugins/jetpack/_inc/build/custom-css/custom-css/js/css-editor.min.js

```diff
@@ -0,0 +1 @@
+!function(){var e,s,o,c,t;e=jQuery,c=function(){s.height(o.height()-s.offset().top-250)},t=function(){s=e("#safecss"),o=e(window),postboxes.add_postbox_toggles("editcss"),c(),e("#safecssform").on("click","#preview",(function(e){e.preventDefault(),document.forms.safecssform.target="csspreview",document.forms.safecssform.action.value="preview",document.forms.safecssform.submit(),document.forms.safecssform.target="",document.forms.safecssform.action.value="save"}))},window.onresize=c,addLoadEvent(t),jQuery((function(e){e(".edit-preprocessor").bind("click",(function(s){s.preventDefault(),e("#preprocessor-select").slideDown(),e(this).hide()})),e(".cancel-preprocessor").bind("click",(function(s){s.preventDefault(),e("#preprocessor-select").slideUp((function(){e(".edit-preprocessor").show(),e("#preprocessor_choices").val(e("#custom_css_preprocessor").val())}))})),e(".save-preprocessor").bind("click",(function(s){s.preventDefault(),e("#preprocessor-select").slideUp(),e("#preprocessor-display").text(e("#preprocessor_choices option:selected").text()),e("#custom_css_preprocessor").val(e("#preprocessor_choices").val()).change(),e(".edit-preprocessor").show()})),e(".edit-css-mode").bind("click",(function(s){s.preventDefault(),e("#css-mode-select").slideDown(),e(this).hide()})),e(".cancel-css-mode").bind("click",(function(s){s.preventDefault(),e("#css-mode-select").slideUp((function(){e(".edit-css-mode").show(),e("input[name=add_to_existing_display][value="+e("#add_to_existing").val()+"]").attr("checked",!0)}))})),e(".save-css-mode").bind("click",(function(s){s.preventDefault(),e("#css-mode-select").slideUp(),e("#css-mode-display").text("true"===e("input[name=add_to_existing_display]:checked").val()?"Add-on":"Replacement"),e("#add_to_existing").val(e("input[name=add_to_existing_display]:checked").val()),e(".edit-css-mode").show()}))}))}();
```

Comment:
:warning: **Warning( severity 5 )**: File appears to be minified and cannot be processed (*Internal.Tokenizer.Exception*).